tests/upgrades-images-signed/upgrade_test.go

```diff
@@ -15,15 +15,44 @@ var _ = Describe("cOS Upgrade tests - Images signed", func() {
 	})
 
 	AfterEach(func() {
+		// Try to gather mtree logs
+		s.GatherLog("/tmp/image-mtree-check.log")
+		s.GatherLog("/tmp/luet_mtree_failures.log")
+		s.GatherLog("/tmp/luet_mtree.log")
 		s.Reset()
 	})
 	Context("After install", func() {
 		When("upgrading", func() {
-			It("fails if verify is enabled on an unsigned version", func() {
-				// Using releases-amd64 as those images are not signed
```

Comment:
I wonder, in this way we aren't checking anymore if we do _fail_ instead when an image is unsigned, but we have the verification turned on?

Reply:
oh good point!

Reply:
I think for sake of testing here, we could even try to upgrade to a `alpine` image, as it will try to pull the metadata file from it, it should fail. That would just test that we are trying to fetch the meta, but not the upgrade itself.

Indeed it's tricky - one way to do this is to push a manual image with a metadata file which have some entries corrupted, and hardcoding that image in the tests

Reply:
Actually I think 0.5.0 is not signed, or at least not properly, so we can use that to test for failure!

Reply:
done,test updated!